android/src/main/java/com/revenuecat/purchases/hybridcommon/common.kt

```diff
@@ -194,9 +194,9 @@ fun logIn(
 ) {
```

Comment:
we should also rename `restoreTransactions` -> `restorePurchases`, but that could be a separate PR